index.html

```diff
@@ -0,0 +1,257 @@
+            <div class="multiple-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- PROFESSION ART PRINTING -->
+    <section class="professional_art_printing-section">
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+
+        <div class="professional_art_printing-section-items">
+            <div class="art_print_container">
+                <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                <p class="art_print_paragraph">
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. Has been the industry's standard.
+                </p>
+                <ul class="art_print_btn_container">
+                    <li><button class="transparent-btn">HTML</button></li>
+                    <li><button class="transparent-btn">Bootstrap</button></li>
+                    <li><button class="transparent-btn">Ruby</button></li>
+                </ul>
+            </div>
+            <div class="art-main-btn-container">
+                <button class="primarybtn">See Project</button>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Please consider wrapping all these project items into one section, which is the work section. All these project items should not use `<section>` but `div` (or `ul` and `li`). This will make your code look much cleaner 🧼 and maintainable 😄